recipes/zarray/bld.bat

```diff
@@ -0,0 +1,8 @@
+cmake -G "NMake Makefiles" -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% -DCMAKE_INSTALL_LIBDIR=lib %SRC_DIR%
+if errorlevel 1 exit 1
+
+nmake
+if errorlevel 1 exit 1
+
```

Comment:
Are there any `make check/test` or `ctest` that we can run here?

Reply:
This was mostly copied from other xtensor packages, maybe @JohanMabille knows?

Reply:
Merging this, you can enable tests, if there are any, on the feedstock later.